### PR TITLE
UOE-7335 : Set Prebid Meta Object to Log

### PR DESCRIPTION
**Pull request by pm-saurabh-narkhede**
# Description

Please add change description or link to ticket, docs, etc.

# Checklist:

- [ ] PR commit list is unique (rebase/pull with the origin branch to keep master clean).
- [ ] JIRA number is added in the PR title and the commit message.
- [ ] Updated the `header-bidding` repo with appropiate commit id.
- [ ] Documented the new changes.

For Prebid upgrade, refer: https://inside.pubmatic.com:8443/confluence/display/Products/Prebid-server+upgrade

### CODE REVIEW

File: adapters/pubmatic/pubmatic.go [hunk 1]
```diff
@@ -555,6 +553,11 @@ func (a *PubmaticAdapter) MakeBids(internalRequest *openrtb2.BidRequest, externa
 				if bidExt.VideoCreativeInfo != nil && bidExt.VideoCreativeInfo.Duration != nil {
 					typedBid.BidVideo.Duration = *bidExt.VideoCreativeInfo.Duration
 				}
+				//prepares ExtBidPrebidMeta with Values got from bidresponse
+				typedBid.BidMeta = prepareMetaObject(bid, bidExt, sb.Seat)
```
Comment (pm-nilesh-chate) [inline adapters/pubmatic/pubmatic.go:557]: add test case to assert bidExt is always `non-nil` to `prepareMetaObject()`

Comment (pm-saurabh-narkhede) [reply adapters/pubmatic/pubmatic.go:557]: Added
